sharness/t0055-service-api-log.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This is using sharness to test something that is not CLI dependent. Use a simple test in `api/rest/` to check that the log file is created.